### ISSUE TITLE
Distilbert: Enable TTNN support

### ISSUE THREAD

**Issue by Sudharsan-V** [tenstorrent/tt-metal#9288]
Enable support for Distilbert QA model using TTNN ops.

**Comment by Sudharsan-V** [comment]
Reference and ttnn implementation for distilbert QA model is completed.
In ttnn implementation 2 PyTorch ops are used (softmax and masked fill).
Reference Implementation [#9413](https://github.com/tenstorrent/tt-metal/pull/9413)
TTNN implementation [commit](https://github.com/tenstorrent/tt-metal/compare/main...sudharsan/distilbert)

**Comment by saichandax** [comment]
Closing this. Tracked in #13327
